lib/linguist/languages.yml

```diff
@@ -2708,6 +2708,15 @@ Lean:
   tm_scope: source.lean
   ace_mode: text
   language_id: 197
+Ledger:
+  type: data
+  extensions:
+  - ".hledger"
+  - ".journal"
+  - ".ledger"
```

Comment:
Shouldn't `.ledger` be the primary extension? If so, it should come first (which is an exception to our usual rule of keeping extensions alphabetised).

Reply:
This is corrected. See https://github.com/github/linguist/pull/4643/commits/40200315c3af5f6b3c4f14eb100fe13a46352649